libpod/container_api.go

```diff
@@ -830,8 +830,15 @@ func (c *Container) Refresh(ctx context.Context) error {
 	return nil
 }
 
+// ContainerCheckpointOptions is a struct used to pass the parameters
+// for checkpointing to corresponding functions
+type ContainerCheckpointOptions struct {
+	Keep        bool
```

Comment:
I think lint will yell at you to provide docs on the fields within the struct

Reply:
I always forget to run lint, thanks for the reminder. I just ran it but the only warnings I get on my tree are:
```
libpod/networking_linux.go:147:13:warning: unused variable or constant invalid operation: syncR (variable of type *os.File) has no field or method SetDeadline (varcheck)
libpod/networking_linux.go:153:7:warning: unused variable or constant IsTimeout not declared by package os (varcheck)
libpod/networking_linux.go:147:13:warning: unused struct field invalid operation: syncR (variable of type *os.File) has no field or method SetDeadline (structcheck)
libpod/networking_linux.go:153:7:warning: unused struct field IsTimeout not declared by package os (structcheck)
```
So it seems okay (from the linter point of view). I was inspired by `ContainerCommitOptions` which also does not trigger a warning from the linter (without documenting the fields).

Reply:
Huh. If the linter's fine with it, I think you're set.